File: src/main/core/com/hrs/Kloping/java/services/impl/BaseServiceImpl.java
package com.hrs.Kloping.java.services.impl;

import com.hrs.Kloping.java.services.BaseService;
import com.hrs.MySpringTool.annotations.Entity;


/**
 * 在这里实现想要做的事
 */
@Entity("base1")
public class BaseServiceImpl implements BaseService {
    @Override
    public Number add(int a, int b) {
        return a + b;
    }
}

File: src/main/core/com/hrs/Kloping/Kotlin/services/impl/BaseServiceImpl.kt
package com.hrs.Kloping.Kotlin.services.impl

import com.hrs.Kloping.Kotlin.services.BaseService
import com.hrs.MySpringTool.annotations.Entity

@Entity("base1")
class BaseServiceImpl : BaseService {
    override fun add(a: Int, b: Int): Number {
        return a + b
    }
}